### ISSUE TITLE
Add Chat feature to website

### ISSUE THREAD

**Issue by riyajha2305** [riyajha2305/Instagram-Clone#16]

**Comment by Ankit62042** [comment]
Hii,I want to work on this issue

**Comment by riyajha2305** [comment]
> Hii,I want to work on this issue

yes sure....you can 👍

**Comment by Ankit62042** [comment]
Hii, May I use external chat feature in this project or I need to create one with js

**Comment by riyajha2305** [comment]
> Hii, May I use external chat feature in this project or I need to create one with js

Yes it’s up to you …. You can use external as well just make sure it works fine with MERN project.

**Comment by riyajha2305** [comment]
> Hii, May I use external chat feature in this project or I need to create one with js

I think the time's already up. Do u still wanna continue? If not do lemme know.

**Comment by dakshgupta2002** [comment]
I want to work on this issue under JWOC 😃

**Comment by riyajha2305** [comment]
> I want to work on this issue under JWOC 😃

Yes sure

**Comment by Devmindtk** [comment]
I would to work on this issue please assign it to me

**Comment by riyajha2305** [comment]
> I would to work on this issue please assign it to me

Sure.Sorry for the late reply.

**Comment by Devmindtk** [comment]
> > I would to work on this issue please assign it to me
> 
> Sure.Sorry for the late reply.

Could you please let me know what type of UI do you want to add for chat feature?

**Comment by riyajha2305** [comment]
> > > I would to work on this issue please assign it to me
> > 
> > 
> > Sure.Sorry for the late reply.
> 
> Could you please let me know what type of UI do you want to add for chat feature?

If it is similar to the original instagram then it would be better
